src/Spectre.libClassifier/Svm.h

```diff
@@ -0,0 +1,42 @@
+/*
+* Svm.h
+* Simple wrapper for OpenCV SVM.
+*
+Copyright 2017 Spectre Team
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#pragma once
+#include <vector>
+#include <opencv2/ml/ml.hpp>
+#include "Spectre.libDataset/IReadOnlyDataset.h"
+#include "Spectre.libClassifier/IClassifier.h"
+#include "Spectre.libClassifier/OpenCvDataset.h"
+#include "Types.h"
+
+namespace Spectre::libClassifier
+{
+class Svm: public IClassifier
```

Comment:
Missing docstring. We must inform, that it only works with OpenCvDataset.

Reply:
done